reference/test/preconditioner/jacobi_kernels.cpp

```diff
@@ -326,6 +327,124 @@ TYPED_TEST(Jacobi, InvertsDiagonalBlocksWithAdaptivePrecision)
 }
 
 
+TYPED_TEST(Jacobi, CanTransposeDiagonalBlocks)
+{
+    using T = typename TestFixture::value_type;
+    using Bj = typename TestFixture::Bj;
+    auto tmp_bj = this->bj_factory->generate(this->mtx);
+    auto bj = gko::as<Bj>(tmp_bj->transpose());
+
```

Comment:
This should be in its own `A` block (Act) since this is what you test.
```suggestion

    auto bj = gko::as<Bj>(tmp_bj->transpose());

```